fast_and_reliable.py

```diff
@@ -0,0 +1,35 @@
+import time
+
+
+def fast_and_reliable(path):
+    """
+    :param path: path of words text file
+    generate list and set of words from path and print
```

Comment:
The function generates a list and a set of the words from the given path's file, and then prints the average time for searching a given word in those data structures.